packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -45,6 +50,10 @@ export class ActiveTracingHelper implements TracingHelper {
     const tracer = trace.getTracer('prisma') as Tracer
 
     engineSpanEvent.spans.forEach((engineSpan) => {
+      if (this.ignoreLayersTypes.map((value) => value.toString()).includes(engineSpan.name)) {
```

Comment:
Should we maybe do the mapping in the constructor instead to avoid allocating a new array each time a new span arrives?

Reply:
Done